### PR TITLE
Submit/settings

### PR DESCRIPTION
**Pull request by Dimava**
Adds Debug settings to the ingame menu
Setting are used only if enableDebugSettings is set to true (in the ingame menu)
Settings are hidden and unchangeable outside of IS_DEBUG builds

### CODE REVIEW

**Comment by Dimava** [comment]
Please test if prod crashes or displays settings - IDK how to build it

File: src/js/profile/application_settings.js [hunk 1]
```diff
@@ -119,6 +121,18 @@ export const allApplicationSettings = [
     new BoolSetting("offerHints", categoryGame, (app, value) => {}),
 ];
 
+/** @type {Array<BaseSetting>} */
+export const allDebugSettings = [];
+for (const k in globalConfig.debug) {
+    if (!IS_DEBUG) break;
```
Comment (tobspr) [inline src/js/profile/application_settings.js:127]: Better wrap the whole block in IS_DEBUG

Comment (Dimava) [reply src/js/profile/application_settings.js]: I chose to disable them instead. Request changes again if appropriate.

**Comment by Dimava** [comment]
Leaving it as is for now, good enough for dev feature, not required for prod

**Comment by Dimava** [comment]
I have **force-pushed** rebased branch here

**Comment by tobspr** [comment]
I'll close this PR since its not planned to be merged

**Pull request closed without merging**